app/Repositories/JsonBaseRepository.php

```diff
@@ -44,7 +44,7 @@ public function allGroupedBy($field)
             throw new InvalidFieldException($field);
```

Comment:
you already throw this error inside the getGroupedByMethod, no reasons to throw it here